tests/integration/users/test_pending_users.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+
+
+@pytest.mark.usefixtures("empty_test_db")
+class TestPendingUsers:
+    def test_users_pending(self, client, valid_header, create_unauthorized_user):
+        pending_user = create_unauthorized_user()
+        assert pending_user.role is None
+        assert pending_user.archived is False
```

Comment:
These assertions are not testing the endpoint here. These can be removed as the `create_unauthorized_user` already has its own tests in `codeforpdx/dwellinglybackend/tests/unit/test_user.py` and it does not need to be retested here.